app/models/event.rb

```diff
@@ -118,6 +118,10 @@ def persons_text
     end
   end
 
+  def video_recordings_sorted
+    self.recordings.video.sort_by { |x| (x.language == self.original_language ? 0 : 2) + (x.html5 ? 0 : 1) }
```

Comment:
Ist zwar nicht wirklich schön, funktioniert aber relativ gut. Alternative wäre https://grosser.it/2010/07/30/ruby-true-false-comparison-with/